From b6cb2b47a3813da5df8dbffada284b72d7fe099e Mon Sep 17 00:00:00 2001
From: Simon Kowallik <github@simonkowallik.com>
Date: Sat, 18 May 2019 13:10:08 +0200
Subject: [PATCH 1/2] fix #554: update tests, Pipfile, Pipfile.lock for
 werkzeug>=0.15.1

- update test_httpbin.py to reflect new behaviour of werkzeug
- require werkzeug>=0.15.1
---
 Pipfile         | 2 +-
 Pipfile.lock    | 6 +++---
 test_httpbin.py | 2 --
 3 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/test_httpbin.py b/test_httpbin.py
index b7104ff..ea51ae8 100755
--- a/test_httpbin.py
+++ b/test_httpbin.py
@@ -148,7 +148,6 @@ def test_get(self):
         data = json.loads(response.data.decode('utf-8'))
         self.assertEqual(data['args'], {})
         self.assertEqual(data['headers']['Host'], 'localhost')
-        self.assertEqual(data['headers']['Content-Length'], '0')
         self.assertEqual(data['headers']['User-Agent'], 'test')
         # self.assertEqual(data['origin'], None)
         self.assertEqual(data['url'], 'http://localhost/get')
@@ -162,7 +161,6 @@ def test_anything(self):
         data = json.loads(response.data.decode('utf-8'))
         self.assertEqual(data['args'], {})
         self.assertEqual(data['headers']['Host'], 'localhost')
-        self.assertEqual(data['headers']['Content-Length'], '0')
         self.assertEqual(data['url'], 'http://localhost/anything/foo/bar')
         self.assertEqual(data['method'], 'GET')
         self.assertTrue(response.data.endswith(b'\n'))
